src/crate/theme/rtd/conf/__init__.py

```diff
@@ -98,6 +99,15 @@
     'crate-docs-theme': ('https://crate-docs-theme.readthedocs.io/en/latest/', None),
 }
 
+# Configure OpenGraph extension
+ogp_site_url = "https://crate.io/"
+ogp_description_length = 300
+ogp_site_name = "Crate.io documentation"
```

Comment:
should we change this to "CrateDB documentation"? people are probably searching for cratedb rather than crate.io

Reply:
Good point!